tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -54,3 +74,20 @@ def test_get(self):
         self.assertIs(list_of_aspect.get('Body.Length'),
                       Metadata.CommitMessage.Body.Length)
         self.assertIsNone(list_of_aspect.get(Metadata))
+
+    def test_get_excludes(self):
+        self.assertIs(self.aspectlist_excludes.get(
+                      Metadata.CommitMessage.Shortlog.ColonExistence),
+                      Metadata.CommitMessage.Shortlog.ColonExistence)
+        self.assertIsNone(self.aspectlist_excludes.get(
+                          Metadata.CommitMessage.Shortlog.TrailingPeriod))
+        self.assertIsNone(self.aspectlist_excludes.get(
+                          Metadata.CommitMessage.Body.Existence))
+
+        self.assertEqual(self.instancelist_excludes.get(
+                      Metadata.CommitMessage.Shortlog.ColonExistence),
+                      Metadata.CommitMessage.Shortlog.ColonExistence('py'))
```

Comment:
I don't think this is a complex call because it just a method call with 1 arg (albeit with very long names). Also every other test have the same pattern.

But I'll try shorten the aspect name to make it easier to read.